pkg/client/restclient/client.go

```diff
@@ -87,7 +87,7 @@ func NewRESTClient(baseURL *url.URL, versionedAPIPath string, config ContentConf
 		config.GroupVersion = &unversioned.GroupVersion{}
 	}
 	if len(config.ContentType) == 0 {
-		config.ContentType = "application/json"
+		config.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
This will mean that kubectl 1.3 will fail by default when contacting a Kube 1.2 server on POSTs and PUTs.